src/test/daml/Daml/Finance/Instrument/Bond/Test/Util.daml

```diff
@@ -51,15 +54,46 @@ originateCallableBond depository issuer label description observers lastEventTim
       couponSchedule = createPaymentPeriodicSchedule firstCouponDate holidayCalendarIds
         businessDayConvention couponPeriod couponPeriodMultiplier issueDate maturityDate
       callSchedule = couponSchedule
+
```

Comment:
maybe you want to add a test for the case when the creation should fail, i.e., a case where `getClaims` fails

Reply:
Good point, I will do that as part of a separate PR.